web/templates/navbar.hbs

```diff
@@ -1,9 +1,19 @@
 <div class="header">
     <nav class="header-main" id="top_navbar">
         <div class="column-left">
-            <a class="left-sidebar-toggle-button {{#if embedded}}hide-streamlist-toggle-visibility{{/if}}" tabindex="0" role="button">
-                <i class="fa fa-reorder" aria-hidden="true"></i>
-                <span class="left-sidebar-toggle-unreadcount">0</span>
+           <a class="left-sidebar-toggle-button {{#if embedded}}hide-streamlist-toggle-visibility{{/if}}" tabindex="0" role="button">
+           <svg id="leftside-open-dot" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-panel-left-dashed">
+                <rect width="18" height="18" x="3" y="3" rx="2"/>
+                <path d="M9 14v1"/>
+                <path d="M9 19v2"/>
+                <path d="M9 3v2"/>
+                <path d="M9 9v1"/>
+            </svg>
+            <svg id="leftside-close-solid" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-panel-left" style="display: none;">
+                <rect width="18" height="18" x="3" y="3" rx="2"/>
+                <path d="M9 3v18"/>
+            </svg>
```

Comment:
See how `zulip-icon` are used in the codebase. Like `zulip-icon-bot` by defining a `bot.svg` file.